Frends.FixedWidthFlatFile.ConvertToJSON/Frends.FixedWidthFlatFile.ConvertToJSON/Definitions/Input.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace Frends.FixedWidthFlatFile.ConvertToJSON.Definitions
+{
+    public class Input
+    {
+        /// <summary>
+        /// Fixed width flat file content
+        /// </summary>
+        [DisplayFormat(DataFormatString = "Expression")]
+        public List<Dictionary<string, dynamic?>> FileContent { get; set; } = new List<Dictionary<string, dynamic?>>();
+        
+        /// <summary>
+        /// Culture format json will be build with
+        /// </summary>
+        [DisplayFormat(DataFormatString = "Text")]
+        public string culture { get; set; } = null;
```

Comment:
Use string.Empty instead of null, since it is a better way to handle empty string values than null. Null can cause exceptions when data is accessed in code, but string.Empty should work fine.